970-ng-commons/contracts/openapi/v1/openapi.yaml

```
@@ -7,70 +7,118 @@ info:
+    | updated    | int64   | none    | Unix timestamp when the resource was last edited in milliseconds                                                           |
     
-    ## Versioning
     
-      The current version of our API is **1.0**.
+    ## Status Codes
+
+      Harness follows conventional HTTP status codes to indicate status of the API request. 
+      Generally, 2xx responses are reserved for a success, while 4xx status codes are reserved for failures. 5xx response implies a server error on Harness' side.
+
+      | Error Code  | Description |
+      |-------------|-------------|
+      | 200         |     OK      |
+      | 201         |   Created   |
+      | 202         |   Accepted  |
+      | 204         |  No Content |
+      | 400         | Bad Request |
+      | 401         | Unauthorized |
+      | 403         | Forbidden |
+      | 412         | Precondition Failed |
+      | 415         | Unsupported Media Type |
+      | 500         | Server Error |
     
-      The version number represents the core API and does not change frequently. The version number changes only if there is a significant departure from the basic underpinnings of the existing API. For example, the version number would change with a system-wide refactoring of core concepts or resources.
-      This version number is represented within the API path. For example, the v1 in `api.harness.io/v1/` is the version number.
+      To view our Error Responses structures, please click [here](#tag/Error-Response).
+
+    ## Versioning
+
+    ### Harness Version
+      The current version of our Beta APIs is **to be announced**. This will be a Date Header format, and will be valid only for our Beta APIs.
     
+    ### Generation
+      All our Beta APIs have a **Generation** that is represented within API path. For example, the v1 in `app.harness.io/v1/`.
+
+      This number represents the core API and will not change frequently. The Generation will change only if there is a significant departure from the basic underpinnings of the existing API. For example, when we have a system-wide refactoring of core concepts or resources.
+
     ## Pagination
-    
-      We use pagination to place limits on the number of responses associated with list endpoints. Pagination is achieved by the use of limit query parameters. The limit defaults to 30. Its maximum value is 100.
-    
-      Link headers are used, which are useful for moving between pages of the overall list.
-    
+
+    We use pagination to place limits on the number of responses associated with list endpoints. Pagination is achieved by the use of limit query parameters. The limit defaults to 30. Its maximum value is 100.
+
+    There are 3 Pagination headers supported in the Response bodies of Paginated APIs.
+      1. X-Total-Elements : Indicates the total number of entries in the Paginated response.
```

Global change (remove space before the colon):
```suggestion
      1. X-Total-Elements: Indicates the total number of entries in a paginated response.
```
(Also use lowercase: "paginated" globally, unless that's the proper noun in some way (unlikely).)